### ISSUE TITLE
[SF][A11y][DatePicker] - In High contrast black on white theme (sap_belize_hcw), the current date outline in date picker fails to meet 7:1

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents#5377]
### **Bug Description**
In High contrast black on white theme (sap_belize_hcw), the current date outline in date picker fails to meet 7:1
![screenshot-2](https://user-images.githubusercontent.com/1751846/174275816-28c1a5af-74e0-4156-9505-aa4b14d4987e.png)

### **Expected Behavior**
UI5 reference:
If you observe there is a white space between the grey background and the purple outline in order for this outline color to meet 7:1 color contrast against white which is the immediate background color. Same must be applied here as well.
![screenshot-1](https://user-images.githubusercontent.com/1751846/174276008-baacb952-806a-4cd1-ba46-7aeee53b454b.png)

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/inputs-datepicker--default-story&globals=theme:sap_belize_hcw;contentDensity:Compact
2. Open the date picker
3. Observe the contrast 

### **Context**
- UI5 Web Components version: 1.4.0
- OS/Platform: macOS
- Browser: Chrom
- Affected component: DatePicker

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Comment by dimovpetar** [comment]
Hello @SAP/ui5-webcomponents-topic-b ,

This issue is also reproducible in the test pages - https://sap.github.io/ui5-webcomponents/main/playground/main/pages/DatePicker/?sap-ui-theme=sap_belize_hcw

Best regards,
Petar

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

A consultation with the visual design colleagues should take place.

Best Regards,
Boyan Rakilovski

**Comment by anuvenkatesh1** [comment]
I have spoken to our Central team expert. She didn't have access to github so pasting her comments here:
"SAP products are supposed to follow the SAP Product Standards as these are our quality measures. The product standard Accessibility also incorporates user experience from affected users and customer feedback. Therefore, it also contains rules for color contrast for high contrast themes also for non-text content, as Anu already wrote and linked. The lack of specific success criteria in external regulations shall not stop us from supporting our users the best we can.

So please enhance the visibility and perceivability of the today marker. The Horizon color palette offers appropriate colors.
As Anu wrote, aligning with the design in UI5 also contributes to a consistent user experience, which in turn supports users with learning issues, memory issues, users who are easily distracted or stressed, to name a few."

If you have further concerns, please reach out to our Central team Accessibility expert Niemann, Gundula <gundula.niemann@sap.com>

**Comment by don-obrien** [comment]
Can we please have a status update on this?  When can we expect a fix?
